aiohttp_sse/__init__.py

```diff
@@ -62,6 +64,10 @@ def __init__(self, *, status=200, reason=None, headers=None):
             # usually not known beforehand.
             self.enable_chunked_encoding()
             return writer
+        else:
+            if request.protocol.transport is None:
```

Comment:
@asvetlov can you please check this line. Is it safe to use of considered private? Also is it good way to check if connection is dead?

Reply:
I've opened aio-libs/aiohttp#3105 for it.
Let's keep the code until the issue will be implemented.

I don't see the coverage, it is the much worse problem.